lib/expo/message.ex

```diff
@@ -147,4 +145,42 @@ defmodule Expo.Message do
       when mod in [Singular, Plural] do
     mod.source_line_number(message, block, default)
   end
+
+  @doc """
+  Merge two messages
+
+  ## Examples
+
+      iex> a = %Expo.Message.Singular{msgid: ["test"], flags: ["one"]}
+      ...> b = %Expo.Message.Singular{msgid: ["test"], flags: ["two"]}
+      ...> Expo.Message.merge(a, b)
+      %Expo.Message.Singular{msgid: ["test"], flags: ["one", "two"]}
+
+      iex> a = %Expo.Message.Singular{msgid: ["test"]}
+      ...> b = %Expo.Message.Plural{msgid: ["test"], msgid_plural: ["tests"]}
+      ...> Expo.Message.merge(a, b)
+      %Expo.Message.Plural{msgid: ["test"], msgid_plural: ["tests"]}
+
+  """
+  @doc since: "0.5.0"
+  @spec merge(a :: Singular.t(), b :: Singular.t()) :: Singular.t()
+  @spec merge(a :: t(), b :: Plural.t()) :: Plural.t()
+  @spec merge(a :: Plural.t(), b :: t()) :: Plural.t()
+  def merge(%mod{} = a, %mod{} = b), do: mod.merge(a, b)
```

Comment:
Don't we have similar logic for merging messages somewhere in the codebase, that we use for `gettext.merge` maybe?

Reply:
Gotcha.